src/control/cmd/dmg/pool.go

```diff
@@ -197,6 +197,10 @@ func (r *PoolExcludeCmd) Execute(args []string) error {
 		return errors.WithMessage(err, "parsing rank list")
 	}
 
+	if len(idxlist) == 0 {
+		idxlist = append(idxlist, 0xFFFFFFFF)
```

Comment:
What if we detect it on the IO server side? Injecting a flag value into the list feels a bit messier to me than just sending an empty list.

Reply:
Sure, that seems reasonable. I'll update the patch.